python/ray/autoscaler/commands.py

```diff
@@ -242,18 +243,15 @@ def get_or_create_head_node(config, config_file, no_restart, restart_only, yes,
                     provider.external_ip(head_node)))
 
     monitor_str = "tail -n 100 -f /tmp/ray/session_*/logs/monitor*"
-    for s in init_commands:
-        if ("ray start" in s and "docker exec" in s
-                and "--autoscaling-config" in s):
-            monitor_str = "docker exec {} /bin/sh -c {}".format(
-                config["docker"]["container_name"], quote(monitor_str))
+    use_docker = bool(config["docker"]["container_name"])
     if override_cluster_name:
         modifiers = " --cluster-name={}".format(quote(override_cluster_name))
     else:
         modifiers = ""
     print("To monitor auto-scaling activity, you can run:\n\n"
-          "  ray exec {} {}{}\n".format(config_file, quote(monitor_str),
-                                        modifiers))
+          "  ray exec {} {}{}{}\n".format(config_file, "--docker "
```

Comment:
Should these be two different commands, e.g. `ray exec` and `ray exec_docker`? At least on the python side, I would want to be able to call `exec_cmd` and `exec_docker_cmd`.

Reply:
Or nevermind, of course I can call `exec_cmd(..., docker=True, ...)` on python too.

Reply:
Sounds good